guides/source/active_storage_overview.md

```diff
@@ -858,15 +858,16 @@ transformations depending on the image's format:
 
 2. If `quality` is not specified, the variant processor's default quality for the format will be used.
 
-The default processor for Active Storage is MiniMagick, but you can also use
-[Vips][]. To switch to Vips, add the following to `config/application.rb`:
+The default processor for Active Storage is now [Vips][], but if using
+configuration defaults below 7.0 it will be MiniMagick. To switch between the
+two, add the following to `config/application.rb`:
```

Comment:
```suggestion
The default processor for Active Storage is [Vips][], but you can also use
MiniMagick. To switch to MiniMagick, add the following to `config/application.rb`:
```
Do we have to mention that prior rails 7 mini magick is default? The docs of the prior versions will already state this

Reply:
Some apps might be using Rails 7 but not yet using `config.load_defaults 7.0`, so only mentioning Vips as the default might also cause confusion.  Perhaps we could sidestep the issue?

```suggestion
Active Storage can use either [Vips][] or MiniMagick as the variant processor.
The default depends on your `config.load_defaults` target version, and the 
processor can be changed by setting [`config.active_storage.variant_processor`](
configuring.html#config-active-storage-variant-processor). 
```

(Note that [`config.active_storage.variant_processor`](https://edgeguides.rubyonrails.org/configuring.html#config-active-storage-variant-processor) shows the versioned defaults.)